src/test/java/com/wdtinc/mapbox_vector_tile/adapt/jts/model/Extent.java

```diff
@@ -0,0 +1,7 @@
+package com.wdtinc.mapbox_vector_tile.adapt.jts.model;
+
+public final class Extent {
```

Comment:
Don't need a whole new class for this, see MvtLayerParams#Default.extent. Could just put the DEFAULT value here in that class as DEFAULT_EXTENT and sub it in for 4096.

Reply:
> What's the use case for this? You want to retain the original tile extent data so that you can write it back out the same?

I do have a requirement of intercepting the tiles and removing some feature on the fly, tiles are not always 4096 extent, retrieving the current extent is necessary.